bliss/predict.py

```diff
@@ -23,19 +123,45 @@ def predict(cfg, image, background):
         pred = encoder.encode_batch(batch)
         est_cat = encoder.variational_mode(pred)
 
-    print("{} light sources detected".format(est_cat.n_sources.item()))
+    if (cfg.predict.plot.ifPlot) & (true_plocs is not None):
```

Comment:
Would `and` have worked here rather than `&`? If so, I'd prefer `and` because its more common